### PR TITLE
Add SpaceCrate.yaml

### PR DESCRIPTION
**Pull request by adegans**
New plugin for ES 0.9.16.1+

### CODE REVIEW

File: manifests/spacecrate.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: SpaceCrate
+authors: Arnan de Gans
+homepage: https://github.com/adegans/SpaceCrate
+license: GPL-3.0
+version: v0.2
+shortDescription: A trade oriented mod for Endless Sky.
+description: SpaceCrate adds planet X-283N1 in the Alpha Centauri system. A rapidly developing world
```
Comment (quyykk) [inline manifests/spacecrate.yaml:7]:
```suggestion
description: SpaceCrate adds planet X-283N1 in the Alpha Centauri system. A rapidly developing world
```

File: manifests/spacecrate.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: SpaceCrate
+authors: Arnan de Gans
+homepage: https://github.com/adegans/SpaceCrate
+license: GPL-3.0
+version: v0.2
+shortDescription: A trade oriented mod for Endless Sky.
+description: SpaceCrate adds planet X-283N1 in the Alpha Centauri system. A rapidly developing world 
+  obsessed with trade and exports. Run by CEO Arnan de Gans this obsession attracts workers from
```
Comment (quyykk) [inline manifests/spacecrate.yaml:8]:
```suggestion
  obsessed with trade and exports. Run by CEO Arnan de Gans this obsession attracts workers from
```

File: manifests/spacecrate.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: SpaceCrate
+authors: Arnan de Gans
+homepage: https://github.com/adegans/SpaceCrate
+license: GPL-3.0
+version: v0.2
+shortDescription: A trade oriented mod for Endless Sky.
+description: SpaceCrate adds planet X-283N1 in the Alpha Centauri system. A rapidly developing world 
+  obsessed with trade and exports. Run by CEO Arnan de Gans this obsession attracts workers from 
+  many planets and results in lucrative trade.
+url: https://github.com/adegans/SpaceCrate/archive/refs/tags/$version.zip
```
Comment (quyykk) [inline manifests/spacecrate.yaml:10]:
```suggestion
url: https://github.com/adegans/SpaceCrate/archive/refs/tags/v0.2.zip
```

File: manifests/spacecrate.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+iconUrl: https://github.com/adegans/SpaceCrate/blob/main/icon.png
```
Comment (quyykk) [inline manifests/spacecrate.yaml:11]:
```suggestion
iconUrl: https://github.com/adegans/SpaceCrate/raw/v0.2/icon.png
```

File: manifests/spacecrate.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+autoupdate:
+  type: tag
+  update_url: https://github.com/adegans/SpaceCrate
+  url: https://github.com/adegans/SpaceCrate/archive/refs/tags/$version.zip
+  iconUrl: https://raw.githubusercontent.com/adegans/SpaceCrate/$version/icon.png
```
Comment (quyykk) [inline manifests/spacecrate.yaml:16]:
```suggestion
  iconUrl: https://github.com/adegans/SpaceCrate/raw/$version/icon.png
```

**Review by quyykk — COMMENTED** [review]
Should fix all the errors

**Comment by adegans** [comment]
That's a very pedantic filter/check :( But suggested edits look good to me... Thanks!

**Review by quyykk — APPROVED** [review]

**Pull request merged**